editing/fx/passes/pact/harmonize.py

```diff
@@ -455,13 +445,11 @@ def __init__(self):
                                                                    make_module_fn=self.make_dummy_bn,
                                                                    name="BIASED_CONV_AND_ACT")
 
-
-
 class HarmonizePACTNetPass(SequentialPass):
-    def __init__(self, **kwargs):
+    def __init__(self, symbolic_trace: callable = PACT_symbolic_trace, **kwargs):
```

Comment:
The type annotation looks wrong. I think it should be Callable[[Union[nn.Module, fx.GraphModule]], fx.GraphModule] or similar.